versions/kruise/1.3.0/templates/apps.kruise.io_statefulsets.yaml

```diff
@@ -0,0 +1,975 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.7.0
+  creationTimestamp: null
+  name: statefulsets.apps.kruise.io
+spec:
```

Comment:
No `conversion` in this spec. These CRDs should be generated by `./bin/kustomize build config/default` from kruise repo, not just copied from `config/crd/bases`.